src/languages/dotnet.md

```diff
@@ -0,0 +1,78 @@
+# .NET
+
+Platform.sh supports deploying .NET applications by allowing developers to define a build process and pass its variables to the .NET build environment.
+
+## Supported versions
+
+* 2.0
+* 2.1
+* 2.2
```

Comment:
Shouldn't this be using the new registry now? :smile:

Reply:
Absolutely, haven't gotten to pulling in the supportedHTML element yet on any of them.

Reply:
Oh!  OK, I thought we'd done that part.